scripts/check-url-locale.js

```diff
@@ -244,16 +239,17 @@ async function main() {
 
   let exitCode = 0;
 
+  console.info(`Crawling ${files.length} files…`);
```

Comment:
Why move this down, when the crawling happens above?

Reply:
I'll see if `no_tty` works and if so, will revert these changes

Reply:
Resolved in https://github.com/mdn/translated-content/pull/26482/commits/31d2eb49013fff33114608e6bb32abbdd16de0e9